examples/pod.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: test
+spec:
+  containers:
+  - name: main
+    image: busybox
+    command:
+    - /bin/sh
+    - -c
+    - 'while true; do sleep 1000; done'
+    volumeMounts:
+    - mountPath: /data
+      name: data
+  volumes:
+  - name: data
+    csi:
+      driver: image.csi.k8s.io
+      volumeAttributes:
+        image: kfox1111/misc:test
```

Comment:
Should we use some public/common image here?

Reply:
Yeah, I was wondering about that too...

What should we do? I could add an example image in the pr and add it to the release scripts too. Would we need a second docker repo for that to push to?

Reply:
Would some popular image like nginx be sufficient? It's not really meant for this purpose but I don't think it's worth creating another repo for an example image

Reply:
I think this issue is resolved for now. Can we merge this pr?